components/addContact/index.jsx

```diff
@@ -52,14 +74,10 @@ export function handleSubmit({ alertError, alertSuccess, fetch, webId }) {
     if (profileError) alertError(profileError);
     if (profile) {
       const contact = { webId: iri };
-      const name = getStringNoLocale(profile.dataset, foaf.name);
-      const fn = getStringNoLocale(profile.dataset, vcard.fn);
-
-      if (name) contact.fn = name;
-      if (fn) contact.fn = fn;
-
+      contact.fn =
```

Comment:
Maybe lets use `fetchProfile` instead, so we have a "standard" way of handling names, in a prepared object.